### PR TITLE
Inputs to render function are converted to float #4681

### PR DESCRIPTION
**Pull request by aniketk21**
@eteq @patti `out` and `coords` are converted to float. Please suggest any changes. #4681

### CODE REVIEW

**Comment by aniketk21** [comment]
@eteq @patti Please review.

**Comment by bmorris3** [comment]
Looks good. I'm going to wait on a response from @eteq and @patti to double check that the `coords` array should be forced to an array.

**Comment by patti** [comment]
@aniketk21 Looks good to me, but the `if` statements are redundant with those below and can be merged. I'm effectively off the grid for the next month, so I'm going to be slow to reply. If @eteq is happy, I'm happy.

**Comment by aniketk21** [comment]
@patti I'll merge them.

**Comment by aniketk21** [comment]
Should I update the docs and add a changelog?

**Comment by pllim** [comment]
Maybe a change log.

**Comment by aniketk21** [comment]
@pllim Under which version should I add a change log?

**Comment by pllim** [comment]
That is a question for your mentor. Is it @eteq ?

**Comment by aniketk21** [comment]
@eteq Under which version should I add a change log?

**Comment by eteq** [comment]
@aniketk21  - Lets call it 1.2, as I said in #4681.  if it's a change in behavior it normally has to go in the next "major" version.  So it should be mentioned both in the `modeling` section and as an "API change" (it's worth noting that it's a change to reflect the fact that this was the *intent*, but it was never clearly stated as far as I know in the docs, so it's not a bug per se but rather an implementation oversight).

**Comment by aniketk21** [comment]
I'm not sure why the build failed. Maybe it happened because I did `git fetch astropy` in the `render-4681` branch by mistake. What should I do now?

**Comment by bsipocz** [comment]
@aniketk21  - There are conflicts, thus the ci can't start, you need to rebase your branch on top of the latest master.

File: CHANGES.rst [hunk 1]
```diff
@@ -46,6 +46,8 @@ New Features
 - ``astropy.modeling``
 
   - Added the fittable=True attribute to the Scale and Shift models with tests. [#4718]
+  - Inputs (``coords`` and ``out``) to ``render`` function in ``Model`` are
```
Comment (pllim) [inline CHANGES.rst]: Remove this duplicate change log entry.

**Comment by pllim** [comment]
Also, would be nice to squash the commits into one before merge, given that there are only effectively 2 lines of new codes, which do not justify 4 different commits.

```
git rebase -i HEAD~4
```

File: CHANGES.rst (hunk 1 above)
Comment (aniketk21) [inline CHANGES.rst]: Okay, thanks!

**Comment by aniketk21** [comment]
@pllim I've squashed all commits into one.

**Comment by pllim** [comment]
Thanks, @aniketk21 ! 

Looks good to me as well. So, the final question is, "Are you happy, @eteq ?"

**Comment by astrofrog** [comment]
@aniketk21 - could you rebase again on the latest master version?

**Comment by aniketk21** [comment]
@astrofrog rebased on the latest master.

**Comment by eteq** [comment]
I am happy ;) Merging!

**Pull request merged**